packages/swc-helpers/.dprint.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
Can you revert this?

Reply:
Did you commit this intentionally?

Reply:
I use it to fomat code.
I should add it to the `.npmignore`